sample-apps/celery_sample/monitor.py

```diff
@@ -0,0 +1,7 @@
+from celery import current_app
+
+from judoscale.celery.tasks_monitor import CeleryEventsHandler
+
+
+events_handler = CeleryEventsHandler(current_app)
+events_handler.start_listening()
```

Comment:
So user need to run in addition python `monitor.py`
While metrics are gathered, and reporter is started, there are connections timed out to the `judoscale-python.requestcatcher.com`. See screenshot.
![Screenshot from 2022-11-11 19-42-45](https://user-images.githubusercontent.com/5695572/201403553-44a6afa7-e610-4824-b2b6-4b4aaeceb142.png)
I wonder if the problem is with the ENV var `DYNO.` Because the ENV var `JUDOSCALE_URL` also was not picked up, and that is why I added it in the file judoscale/config,py:

Reply:
Interesting... I did not get that error:


```
(.venv) celery_sample $ python monitor.py
[2022-11-11 21:23:25.083265] TASK-RECEIVED blog.tasks.add (STATE=RECEIVED, UUID=9049af8d-382b-4f14-a32b-c3bcac6eaf94)
INFO - [Judoscale] Starting reporter for process 14079
[2022-11-11 21:23:25.085320] TASK-STARTED blog.tasks.add (STATE=STARTED, UUID=9049af8d-382b-4f14-a32b-c3bcac6eaf94)
[2022-11-11 21:23:30.151912] TASK-SUCCEEDED blog.tasks.add (STATE=SUCCESS, UUID=9049af8d-382b-4f14-a32b-c3bcac6eaf94)
```

But I also don't see any log of anything being reported.